### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by RooRay**
Improves the wording of the README.md and also includes the Netlify deploy link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# docs
+# Docs
```
Comment (eartharoid) [inline README.md:1]:
```suggestion
# Documentation
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/f1919fde-0ef7-4b52-884f-870e354f74e4/deploy-status)](https://app.netlify.com/sites/discordtickets/deploys)
+[![Netlify Build Status](https://api.netlify.com/api/v1/badges/f1919fde-0ef7-4b52-884f-870e354f74e4/deploy-status)](https://app.netlify.com/sites/discordtickets/deploys)
 
-This is the source for the Discord Tickets main/documentation website.
-You can view the live site at [discordtickets.app](https://discordtickets.app).
+This is the source code for Discord Tickets fourth iteration documentation.
+You can view the latest build of the V4 site [on Netlify](https://v4--discordtickets.netlify.app/).
```
Comment (eartharoid) [inline README.md:6]: Unnecessary and will soon be reverted so no point changing

**Review by eartharoid — CHANGES_REQUESTED** [review]
~~This is a very hacktoberfest PR.~~

The link and text change for v4 is unnecessary as the v4 branch will soon become main.
Also, it's not really source code because it's markdown (markup) not code.

The image alt-text and title changes are ok (although if you're going to make it a proper title rather than just the repo name I'd prefer `Documentation`).

Would you like to add to the end of the contributing section that the live preview will not look exactly the same as the production site due to different mkdocs versions & configs?

File: README.md (hunk 2 above)
Comment (RooRay) [reply README.md:6]: ok

**Comment by RooRay** [comment]
> Would you like to add to the end of the contributing section that the live preview will not look exactly the same as the production site due to different mkdocs versions & configs?

I'll add that once I figure out how to with GitHub's Web GUI 🙃

**Comment by RooRay** [comment]
Made a compromise of including info about V4 since there'll be a V3 branch and excluded mentioning of a V4 build since that'd be the production site. Also added the disclaimer as per your request

**Comment by RooRay** [comment]
After some back and forth on Discord I remade it based on Earth's suggestions

File: README.md [hunk 3]
```diff
@@ -1,12 +1,27 @@
...
 
 ## Contributing
 
-For simple changes, you can easily fork, edit, open a PR, and see the result with the PR deploy preview.
-If you want to preview locally, clone and then run `pip install -r requirements.txt` and `mkdocs serve`.
+For simple changes, you can easily create a fork, edit the files you want to change, open a PR, and see the result with the PR deploy preview from the Netlify bot.
+
+If you want to preview locally, clone the repo and then run `pip install -r requirements.txt` and `mkdocs serve`.
+
+## Disclaimer
```
Comment (eartharoid) [inline README.md:17]: Should be H3 but also doesn't need to be a heading at all.

**Review by eartharoid — CHANGES_REQUESTED** [review]

**Review by eartharoid — APPROVED** [review]

**Pull request merged**